content/tutorial/tutorial.md

```diff
@@ -1170,41 +1177,41 @@ We will also replace reading `this.state.history` with `this.state.history.slice
   }
 ```
 
-Finally, we will modify the Game component's `render` method from always rendering the last move to rendering the currently selected move according to `stepNumber`:
+अंत में, हम Game कौम्पोनॅन्ट के `render` मेथड को हमेशा अंतिम चरण प्रदान करने से लेकर वर्तमान में चुने गए कदम को `stepNumber` के अनुसार रेंडर करेंगे।
 
 ```javascript{3}
   render() {
     const history = this.state.history;
     const current = history[this.state.stepNumber];
     const winner = calculateWinner(current.squares);
 
-    // the rest has not changed
+    // और इसके आगे कोई चाल बाकि नहीं है
 ```
 
-If we click on any step in the game's history, the tic-tac-toe board should immediately update to show what the board looked like after that step occurred.
+यदि हम गेम के history के किसी भी चरण पर क्लिक करते हैं, तो टिक-टैक-टो Board को तुरंत यह दिखाने के लिए अपडेट करना चाहिए कि Board उस चरण के बाद कैसा दिखता था।
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**
 
-### Wrapping Up {#wrapping-up}
+### समाप्त करें {#wrapping-up}
```

Comment:
Though this is okay, but we could probably have something better like antim steps (in hindi) or something?

Reply:
`अंतिम स्टेप्स` this sounds better. let me update this